src/hotspot/share/runtime/sharedRuntime.cpp

```diff
@@ -636,27 +636,27 @@ JRT_ENTRY(void, SharedRuntime::notify_jvmti_object_alloc(oopDesc* o, JavaThread*
   current->set_vm_result(h());
 JRT_END
 
-JRT_ENTRY(void, SharedRuntime::notify_jvmti_mount(oopDesc* vt, jboolean hide, jboolean first_mount, JavaThread* current))
+JRT_ENTRY(void, SharedRuntime::notify_jvmti_vthread_start(oopDesc* vt, jboolean dummy, JavaThread* current))
```

Comment:
Maybe rename dummy to hide and just assert is false in this case and true for the vthread_end case?

Reply:
Good suggestion. Thank you.